templates/todo/api/csharp-cosmos-sql/TodoEndPointsExt.cs

```diff
@@ -0,0 +1,143 @@
+﻿using Microsoft.AspNetCore.Http.HttpResults;
+using Microsoft.AspNetCore.Mvc;
+using SimpleTodo.Api;
+
+namespace Todo.Api
+{
+    public static class TodoEndpointsExt
+    {
+        public static void MapTodoEndpoints(this WebApplication app)
+        {
+
+            app.MapGet("/lists", async (ListsRepository _repository, [FromQuery] int? skip = null, [FromQuery] int? batchSize = null) =>

```

Comment:
No need for the explicit `[FromQuery]` attribute here. It's implied for parsable parameter types that don't have an associated route parameter.